src/dev/flang/fe/FrontEnd.java

```diff
@@ -112,7 +112,7 @@ public FeatureName featureName()
    * The library modules loaded so far.  Maps the module name, e.g. "base" to
    * the corresponding LibraryModule instance.
    */
-  private TreeMap<String, LibraryModule> _modules = new TreeMap<>();
+  public TreeMap<String, LibraryModule> _modules = new TreeMap<>();
```

Comment:
If you make this `public`, you should at least make it `final`. 

Better would be to keep this `private` and add a method `modules()` that returns `_modules.values()`.